nfs_types.go

```diff
@@ -103,23 +104,30 @@ type NFSExportModify struct {
 	// NFS enforced security type for users accessing an NFS Export.
 	MinSecurity string `json:"min_security,omitempty"`
 	// Hosts with no access to the NFS export or its snapshots.
-	NoAccessHosts []string `json:"no_access_hosts,omitempty"`
+	// Empty list is a valid value, so omitempty should not be used.
+	NoAccessHosts []string `json:"no_access_hosts"`
 	// Hosts with read-only access to the NFS export and its snapshots.
-	ReadOnlyHosts []string `json:"read_only_hosts,omitempty"`
+	// Empty list is a valid value, so omitempty should not be used.
+	ReadOnlyHosts []string `json:"read_only_hosts"`
 	// Hosts with read-only and read-only for root user access to the NFS Export and its snapshots.
-	ReadOnlyRootHosts []string `json:"read_only_root_hosts,omitempty"`
+	// Empty list is a valid value, so omitempty should not be used.
+	ReadOnlyRootHosts []string `json:"read_only_root_hosts"`
 	// Hosts with read and write access to the NFS export and its snapshots.
-	ReadWriteHosts []string `json:"read_write_hosts,omitempty"`
+	// Empty list is a valid value, so omitempty should not be used.
+	ReadWriteHosts []string `json:"read_write_hosts"`
 	// Hosts with read and write and read and write for root user access to the NFS Export and its snapshots.
-	ReadWriteRootHosts []string `json:"read_write_root_hosts,omitempty"`
+	// Empty list is a valid value, so omitempty should not be used.
+	ReadWriteRootHosts []string `json:"read_write_root_hosts"`
 	// Specifies the user ID of the anonymous account.
-	AnonymousUID int32 `json:"anonymous_UID,omitempty"`
+	// Zero ID is a valid value, so omitempty should not be used.
+	AnonymousUID int32 `json:"anonymous_UID"`
```

Comment:
I am not sure if one can unset _AnonymousUID_ and _GID_. If that is not allowed, let's not remove '_omitempty_'.